diff -up comical-0.8/src/ComicalApp.cpp~ comical-0.8/src/ComicalApp.cpp
--- comical-0.8/src/ComicalApp.cpp~	2008-09-27 20:48:34.000000000 +0200
+++ comical-0.8/src/ComicalApp.cpp	2008-09-27 20:48:34.000000000 +0200
@@ -25,6 +25,7 @@
  *                                                                         *
  ***************************************************************************/
 
+#include <wx/icon.h>
 #include "ComicalApp.h"
 #include "ComicalFrame.h"
 #include <wx/log.h>
