--- comical-0.8/src/ComicBookZIP.cpp.BAD	2006-04-24 20:41:07.000000000 -0500
+++ comical-0.8/src/ComicBookZIP.cpp	2006-04-24 20:41:30.000000000 -0500
@@ -54,7 +54,7 @@
 	do {
 		unzGetCurrentFileInfo(ZipFile, fileInfo, namebuf, 1024, NULL, 0, NULL, 0);
 		page = wxString::FromAscii(namebuf);
-		if(page.Right(5).Upper() == wxT(".JPEG") || page.Right(4).Upper() == wxT(".JPG") ||
+		if(page.Right(5).Upper() == wxT(".JPEG") || page.Right(4).Upper() == wxT(".JPG") || page.Right(4).Upper() == wxT(".JPE") || 
 		page.Right(4).Upper() == wxT(".GIF") ||
 		page.Right(4).Upper() == wxT(".PNG"))
 			Filenames->Add(page);
--- comical-0.8/src/ComicBookRAR.cpp.BAD	2006-04-24 20:41:56.000000000 -0500
+++ comical-0.8/src/ComicBookRAR.cpp	2006-04-24 20:42:15.000000000 -0500
@@ -67,7 +67,7 @@
 #else
 		page = wxString(header.FileName);
 #endif
-		if(page.Right(5).Upper() == wxT(".JPEG") || page.Right(4).Upper() == wxT(".JPG") ||
+		if(page.Right(5).Upper() == wxT(".JPEG") || page.Right(4).Upper() == wxT(".JPG") || page.Right(4).Upper() == wxT(".JPE") ||
 		page.Right(4).Upper() == wxT(".GIF") ||
 		page.Right(4).Upper() == wxT(".PNG"))
 			Filenames->Add(page);
