--- comical-0.8/unrar/makefile.linux.BAD	2006-02-11 10:15:09.000000000 -0600
+++ comical-0.8/unrar/makefile.linux	2006-04-24 20:43:10.000000000 -0500
@@ -7,7 +7,7 @@
 
 # Linux using GCC
 CXX=g++
-CXXFLAGS=-O2 -fPIC
+CXXFLAGS=-O2 -fPIC $(RPM_OPT_FLAGS)
 DEFINES=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
 STRIP=strip
 AR=ar
--- comical-0.8/src/Makefile.BAD	2006-04-24 20:44:53.000000000 -0500
+++ comical-0.8/src/Makefile	2006-04-24 20:45:04.000000000 -0500
@@ -1,5 +1,5 @@
 INCLUDE = -I../unrar -I../unzip
-CFLAGS = -O2 -Wall -pipe
+CFLAGS = -O2 -Wall -pipe $(RPM_OPT_FLAGS)
 CPPFLAGS = `wx-config --cxxflags` $(CFLAGS) -D_UNIX $(INCLUDE)
 
 .SUFFIXES: .cpp .png .h .d .o
