diff -up comical-0.8/Makefile.orig comical-0.8/Makefile
--- comical-0.8/Makefile.orig	2008-10-26 00:23:56.000000000 +0200
+++ comical-0.8/Makefile	2008-10-26 00:24:29.000000000 +0200
@@ -1,5 +1,5 @@
 CC = `wx-config --cxx`
-LDFLAGS = `wx-config --libs` -lunrar -Lunzip -lminiunzip
+LDFLAGS = `wx-config --libs` `pkg-config minizip --libs` -lunrar
 INSTALL = install
 INSTALL_PROGRAM = $(INSTALL)
 prefix = /usr/local
@@ -12,15 +12,12 @@ OBJS = $(patsubst %.cpp,%.o,$(wildcard s
 
 all: comical
 
-comical: $(OBJS) unzip/libminiunzip.a
+comical: $(OBJS)
 	$(CC) -o $@ $(OBJS) $(LDFLAGS)
 
 $(OBJS):
 	$(MAKE) -C src
 
-unzip/libminiunzip.a:
-	$(MAKE) -C unzip
-
 install: comical
 	$(INSTALL_PROGRAM) comical $(DESTDIR)$(bindir)/comical
 
@@ -36,7 +33,6 @@ clean:
 
 distclean:
 	$(MAKE) clean
-	$(MAKE) clean -C unzip
 	rm -f $(DEPS) bin2h src/bin2h.o
 
 .PHONY : install install-strip uninstall clean distclean
diff -up comical-0.8/src/ComicBookZIP.cpp.orig comical-0.8/src/ComicBookZIP.cpp
diff -up comical-0.8/src/Makefile.orig comical-0.8/src/Makefile
--- comical-0.8/src/Makefile.orig	2008-10-26 00:24:36.000000000 +0200
+++ comical-0.8/src/Makefile	2008-10-26 00:25:44.000000000 +0200
@@ -1,4 +1,4 @@
-INCLUDE = -I../unzip
+INCLUDE = `pkg-config minizip --cflags`
 CFLAGS = -O2 -Wall -pipe $(RPM_OPT_FLAGS)
 CPPFLAGS = `wx-config --cxxflags` $(CFLAGS) -D_UNIX $(INCLUDE)
 
